src/.clang-format

```diff
@@ -26,7 +28,6 @@ IndentCaseLabels: false
 IndentFunctionDeclarationAfterType: false
 IndentWidth:     4
 KeepEmptyLinesAtTheStartOfBlocks: false
-Language:        Cpp
```

Comment:
I moved it to the top. But as we are only using cpp it does not matter.

Doc:
> The configuration file can consist of several sections each having different `Language:` parameter denoting the programming language this section of the configuration is targeted at. [...] The first section may have no language set, it will set the default style options for all lanugages.